graphics/embree/files/patch-static-lib.diff

```diff
@@ -0,0 +1,23 @@
+--- kernels/CMakeLists.txt.orig	2020-05-14 01:08:53.000000000 -0400
++++ kernels/CMakeLists.txt	2020-12-08 15:40:57.000000000 -0500
+@@ -221,7 +221,10 @@
+ ENDMACRO()
+ 
+ DISABLE_STACK_PROTECTOR_FOR_INTERSECTORS(${EMBREE_LIBRARY_FILES})
+-ADD_LIBRARY(embree ${EMBREE_LIB_TYPE} ${EMBREE_LIBRARY_FILES} $<TARGET_OBJECTS:algorithms>)
++IF (EMBREE_STATIC_LIB)
++  SET(BUILD_SHARED_LIBS ON)
```

Comment:
heh. pretty intuitive -- if you want a static lib, you turn "build shared libs" on. Jeeps...

Reply:
Yup, gotta love CMake. Although to be fair, I'm trying to make it so that the package builds both at the same time.